### ISSUE TITLE
Make a base stylesheet with variables for common dimensions and colors

### ISSUE THREAD

**Issue by olynch** [ToposInstitute/CatColab#54]
In https://sass-lang.com/, you can create variables and import style sheets into each other. We should make variables for:

1. Colors. We should work out a nice, minimalist, meaningful color scheme. Maybe just a couple well-chosen shades of grey and the Topos blue.
2. Dimensions (so that we aren't using ad-hoc pixel values, we should have some variables like "cell height" and then set other things to multiples of that. For instance, it would be good to have consistent padding across different elements, or border styling, etc.

I think that we don't need to do the same for font: we can just set a global sans, a global serif, and a global monospace. But we should decide what these are.

**Comment by olynch** [comment]
I didn't know "native CSS variables" were a thing! I guess I'm old:
![image](https://github.com/user-attachments/assets/fb79534d-eb49-460f-be95-126a670fd7c3)

**Comment by epatters** [comment]
Yeah, it's great that CSS has native support for variables now! It's a first for me too. I have a few variables defined in the [root stylesheet](https://github.com/ToposInstitute/CatColab/blob/79fe67e6aee0bdba185b0ff85bceb59fbb0503d5/frontend/src/index.css) but it's not very systematic and there are plenty of other places with ad hoc color choices. So the main point of the issue stands.

**Comment by kasbah** [comment]
Just stumbled across this. Point 1 is now a duplicate of #912

**Comment by epatters** [comment]
Thanks for noticing. I'll close this issue, which has a broad scope, and leave it to your judgment, @kasbah, whether you want to create another issue to track point 2.
